lib/providers/http/services/contracts.js

```diff
@@ -47,15 +47,16 @@ class Contracts extends HttpService {
     return data
   }
 
-  async getCallTx(callData, contractPubKey, functionName, args = [], options = {}) {
+  async getCallTx(contractAddress, callData, options = {}) {
     let payload = {
       ...createTxCallParams(options),
-      'caller': options.caller || await this.client.accounts.getPublicKey(),
+      'caller': options.caller,
       'vm_version': options.vmVersion || 1,
-      'contract': contractPubKey,
-      'function': functionName,
-      'arguments': args.join(',')
+      'call_data': callData,
```

Comment:
Why lower_case here? Is this key name required by an external API?

Reply:
yes, its a payload parameter